FrontEnd/src/components/adminPage/detail-view/ModerationEmail.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react';
+import { toast } from 'react-toastify';
+import { Tooltip } from 'antd';
+import useSWR from 'swr';
+import axios from 'axios';
+import css from './ModerationEmail.module.css';
+
+const ModerationEmail = () => {
+    const fetcher = url => axios.get(url).then(res => res.data);
+    const url = `${process.env.REACT_APP_BASE_API_URL}/api/admin/email/`;
+    const { data, mutate } = useSWR(url, fetcher);
+
+    // Define email and setEmail using useState
+    const [email, setEmail] = useState(null);
+    const [error, setError] = useState(null);
```

Comment:
Is the error state necessary in this component? I don't see it being set to anything other than null in the code below.

Reply:
Fixed, thank you